### PR TITLE
Coverage pyemail

### PR DESCRIPTION
**Pull request by chazuttu**
## Summary

This PR adds unit tests for the script **pyemail.py** and also increases its coverage.

- added unit tests for **pyemail.py**
- updated .coveragerc file
- updated workflow file to **copy rece.ini**
- increased coverage to 78%

## Checklist

- [x] Classes, Variables, function and methods logic  ok
- [x] Comments written explaining what the code does
- [x] All python code is PEP8 compliant (run black .)
- [x] No lint issues (run flake8)
- [x] Test coverage with pytest implemented
- [x] Reviewers assigned (at least 1 mentor)

### Python 3
pytest --cov-config=.coveragerc --cov=pyafipws tests/test_pyemail.py
```
========================== test session starts ========================
platform win32 -- Python 3.9.6, pytest-6.2.3, py-1.10.0, pluggy-0.13.1
rootdir: C:\Users\pyafipws, configfile: pytest.ini
plugins: cov-2.12.1, freezegun-0.4.2, html-3.1.1, metadata-1.11.0, mock-3.6.1, vcr-1.0.2
collected 8 items

tests\test_pyemail.py ........                                                                                                                                                                    [100%]

----------- coverage: platform win32, python 3.9.6-final-0 -----------
Name                                                                                       Stmts   Miss  Cover
--------------------------------------------------------------------------------------------------------------
test3\Lib\site-packages\PyAfipWs-3.9.b_2572_-py3.9.egg\pyafipws\__init__.py                    4      0   100%
test3\Lib\site-packages\PyAfipWs-3.9.b_2572_-py3.9.egg\pyafipws\pyemail.py                   173     38    78%
--------------------------------------------------------------------------------------------------------------

=============================== 8 passed, 4 warnings in 1.41s =====================
```

### Python 2
pytest --cov-config=.coveragerc --cov=pyafipws tests/test_pyemail.py
```
============================= test session starts =============================
platform win32 -- Python 2.7.18, pytest-4.6.11, py-1.10.0, pluggy-0.13.1
rootdir: C:\Users\pyafipws, inifile: pytest.ini
plugins: cov-2.12.1, freezegun-0.4.2, html-1.22.1, metadata-1.11.0, mock-2.0.0, vcr-1.0.2
collected 8 items

tests\test_pyemail.py ........                                           [100%]

---------- coverage: platform win32, python 2.7.18-final-0 -----------
Name                                                                                    Stmts   Miss  Cover
-----------------------------------------------------------------------------------------------------------
test2\Lib\site-packages\PyAfipWs-2.7.2572-py2.7.egg\pyafipws\__init__.py                    4      0   100%
test2\Lib\site-packages\PyAfipWs-2.7.2572-py2.7.egg\pyafipws\pyemail.py                   173     38    78%
-----------------------------------------------------------------------------------------------------------

==================== 8 passed, 2 warnings in 1.40 seconds =====================
```

### CODE REVIEW

File: pyemail.py [hunk 1]
```diff
@@ -93,10 +93,10 @@ def Conectar(self, servidor, usuario=None, clave=None, puerto=25):
                 self.smtp.starttls()
             if usuario and clave:
                 # convertir a string (hmac necesita string "bytes")
-                if isinstance(usuario, str):
-                    usuario = usuario.encode("utf8")
-                if isinstance(clave, str):
-                    clave = clave.encode("utf8")
+                # if isinstance(usuario, str):
```
Comment (reingart) [inline pyemail.py]: - [ ] Please leave this 2 if for python2 only

File: pyemail.py [hunk 2]
```diff
@@ -93,10 +93,10 @@ def Conectar(self, servidor, usuario=None, clave=None, puerto=25):
...
+                #     usuario = usuario.encode("utf8")
+                # if isinstance(clave, str):
```
Comment (reingart) [inline pyemail.py]: - [ ] Please leave this if for python2 only (skip inpython3)

**Review by reingart — APPROVED** [review]

File: pyemail.py (hunk 1 above)
Comment (chazuttu) [reply pyemail.py]: - [x] Please leave this 2 if for python2 only

Comment (chazuttu) [reply pyemail.py]: made the required changes

File: pyemail.py (hunk 2 above)
Comment (chazuttu) [reply pyemail.py]: made the required changes

**Review by reingart — APPROVED** [review]

**Pull request merged**